Assets/Scripts/Minion/MinionBehaviour.cs

```diff
@@ -24,6 +24,7 @@ public class MinionBehaviour : Entity
     private Animator animator;
     public Transform target { get; private set; }
     private NavMeshAgent agent;
+    public bool isSturn;
```

Comment:
```suggestion
    public bool isStun;
```
오타입니다.
`isStun`을 의도하신 것 같습니다.